File: src/main/java/it/contrader/service/ExtensionService.java
package it.contrader.service;

import it.contrader.converter.ExtensionConverter;
import it.contrader.dao.ExtensionDAO;
import it.contrader.dto.ExtensionDTO;
import it.contrader.model.Extension;

public class ExtensionService	extends AbstractService<Extension, ExtensionDTO> {
		
		  public ExtensionService(){
			this.dao = new ExtensionDAO();
			this.converter = new ExtensionConverter();
		}
		
}

File: src/service/extension.service.ts
import { Injectable } from '@angular/core';
import { AbstractService } from './abstractservice';
import { HttpClient } from '@angular/common/http';
import { ExtensionDTO } from 'src/dto/extensiondto';

@Injectable({
  providedIn: 'root'
})
export class ExtensionService extends AbstractService<ExtensionDTO> {

  constructor(http: HttpClient) {
    super(http);
    this.type = 'extension';
  }
}
